### PR TITLE
Separator

### PR DESCRIPTION
**Pull request by amawdiin**
I addded a {"separator":""} property for built-in TextProp.nbt() function

### CODE REVIEW

**Review by Nico314159 — CHANGES_REQUESTED** [review]

**Comment by WingedSeal** [comment]
Please implement it on TextProps as well

**Comment by Nico314159** [comment]
> Please implement it on TextProps as well

Yup, my suggested change does that

File: src/jmc/compile/command/builtin_function/load_only.py [hunk 1]
```diff
@@ -1692,10 +1692,12 @@ def inner(arg: str) -> SIMPLE_JSON_BODY:
         "source": ArgType.STRING,
         "path": ArgType.KEYWORD,
         "interpret": ArgType.KEYWORD,
-        "local": ArgType.KEYWORD
+        "local": ArgType.KEYWORD,
+        "separator": ArgType.STRING
```
Comment (Nico314159) [inline src/jmc/compile/command/builtin_function/load_only.py:1696]: I think seperator should be *before* the boolean arguments

**Review by Nico314159 — CHANGES_REQUESTED** [review]

File: src/jmc/compile/command/builtin_function/load_only.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (amawdiin) [reply src/jmc/compile/command/builtin_function/load_only.py:1696]: sure, ill do that

**Review by Nico314159 — APPROVED** [review]
LGTM

**Pull request merged**